monitor/monitoring-dam-generator-core/src/test/java/eu/seaclouds/monitor/monitoringdamgenerator/tests/MonitoringGenerationTest.java

```diff
@@ -28,7 +28,7 @@ public void monitoringInformationGenerationTest() throws Exception {
 
         Map<String, String> parametersTest;
         
-        MonitoringDamGenerator service = new MonitoringDamGenerator(new URL("http://127.0.0.1:8170"));
+        MonitoringDamGenerator service = new MonitoringDamGenerator(new URL("http://127.0.0.1:8170"), new URL("http://127.0.0.1:8083"));
```

Comment:
@adriannieto Any server is started since any communication is realized. The url is taken just as input to properly configure data collectors with the IP and port of T4C, but they will be the DCs themselves to contact T4C sending their configuration after the deployment.